### ISSUE TITLE
Some 1.6+ support code

### ISSUE THREAD

**Issue by alexhulbert** [TechnicPack/TechnicLauncher#295]
I made a wrapper a while ago adding to this. To speed things up a liitle with the 1.6 support, I just wanted to say that you're welcome to use any code from my repo (https://www.github.com/Atex/Migrator) if you want. Its got some code that might help you with the guts of adding 1.6 support such as how to generate and filter the library list from json files and things like that. I didn't know where to put this (since it can't be a PR), so I just made it an issue. if you decide to look at my code, just note that I had to jump through a few hoops to get the login system to work. Since you only pass the username and old token id in your applet stubs, the program decrypts your lastlogin file instead.

**Comment by Salynne** [comment]
Thanks for the offer but if you check out the commits in master I've actually almost got it. I moved over to the yggdrasil authentication and rewrote the whole part that downloads the modpacks, libraries, and launches the game.

We're preparing to promote a beta build as soon as I fix the auto-updater. I made an attempt to launch a beta build today but I had a problem with the relaunch class and I'll finish that up tomorrow.
